yt/frontends/ramses/particle_handlers.py

```diff
@@ -202,6 +202,10 @@ def read_header(self):
             if self.ds._extra_particle_fields is not None:
                 particle_fields += self.ds._extra_particle_fields
 
+        if hvals["nstar_tot"] > 0:
```

Comment:
At this point, it would be wise to check there is no clash with the `extra_particle_fields` argument to prevent the particle birth time to be defined twice?